scenarios/crowdsecurity/http-bf-wordpress_bf.yaml

```diff
@@ -3,7 +3,7 @@ name: crowdsecurity/http-bf-wordpress_bf
 description: "detect wordpress bruteforce"
```

Comment:
Is it possible to rename this scenario, there is "bf" twice in the name.

Reply:
mh not really, I like to keep `<proto>-<attack-type>-<scenario name>` for now